### PR TITLE
Change to Kyle's Main Page

### PR DESCRIPTION
**Pull request by Yoshimitsu45**

### CODE REVIEW

**Comment by Yoshimitsu45** [comment]
@dougwoods3 
@harterj 
@albeanth

**Comment by dougwoods3** [comment]
:shipit:

**Comment by harterj** [comment]
Lookin' good :shipit:

File: users/combsky/index.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ author: Kyle Combs
 Kyle Combs
 ================================
 
+Scribble
```
Comment (dougwoods3) [inline users/combsky/index.md:9]: test comment

**Pull request merged**